docs/conf/prague/2020/sponsors/index.rst

```diff
@@ -0,0 +1,3 @@
+:template: {{year}}/generic.html
```

Comment:
We're getting the Portland cover image, so we should probably make an explicit `:banner:` here.

Reply:
@plaindocs can you hustle this fix quickly? otherwise I say let's keep this for now and change it when we launch the full website, I'm not too troubled by it for now

Reply:
I will look into it today.

Reply:
```suggestion
:banner: _static/conf/images/headers/prague-group.png
```
is last years.

Reply:
Banners are fixed.